utilities/create-mock-data/README.md

```diff
@@ -0,0 +1,35 @@
+# How to run
+
+`dotnet run <number of Enrollees to generate> <number of Sites to generate>`.  Zero is an acceptable parameter.  The target database to generate the Enrollees into is stored in the `appsettings.json` file.  Change as desired.  
+
+
+# ModelFactories and Lookups
+
+Many ModelFactories and Lookups files are copied verbatim from `prime-dotnet-webapi-tests`.  The ModelFactories (`Faker` subclasses) have the `SetBaseRules` call disabled.  Other changes as necessary include:
+
+1. `RuleFor` re: the `Id` of the object is disabled since we let the database assign the `Id`.  For `EnrolleeFactory` we do not disable assigning the `Id` 
+because of some foreign key relationships that need to be established.
```

Comment:
Also, manually setting the Enrollee's Id in the mock data generator means we can't ever run it twice since it will start over at the initialized Id and EF will complain that you are inserting records with duplicate Ids.

Reply:
seems a blank db is needed before running data generation or duplicate PK violation would occur, since the fake enrollee ID starts at 1? maybe smart generating the enrollee ID from the current enrollee count + 1 position in a non-empty database...

Reply:
Addressed the duplicate PK issue in https://github.com/bcgov/moh-prime/pull/1721/commits/3069972ce7fbfc9c1cda2f3cb948122d68942d65

Reply:
> This comes back to the idea of "what are our data factories actually targeting?" that I've mentioned.
> 
> When we are generating data for tests, we need to set up various relationships manually due to the fact that the data represents entities that have been materialized from the database. Since the objects being created didn't _actually_ come from the database, we often have to fill in gaps ourselves to make the data look "real".
> 
> In the case of these data factories, our scope is to create data that will be saved directly into a real database. So we don't need to manually create foreign keys etc. ourselves. If you go
> 
> ```cs
> var enrollee = new Enrollee
> {
>     Certifications = new [] 
>     {
>         new Certification
>         {
>             CollegeCode = 1,
>             LisenseCode = 1,
>             LisenseNumber = "12345" 
>         }
>     }
> }
> ```
> 
> and save it to the database, the FK back to the `Enrollee` will get created automatically. `Certification.EnrolleeId` will be set by EF to whatever Id the `Enrollee` gets assigned.
> 
> This means we don't have to do this:
> 
> ```cs
> // in EnrolleeFactory
> RuleFor(x => x.Certifications, (f, x) => new CertificationFactory(x).GenerateBetween(1, 2));
> 
> // in CertificationFactory
> RuleFor(x => x.Enrollee, f => owner);
> RuleFor(x => x.EnrolleeId, f => owner.Id);
> ```
> 
> we can instead do:
> 
> ```cs
> // in EnrolleeFactory
> RuleFor(x => x.Certifications, f => new CertificationFactory().GenerateBetween(1, 2));
> ```
> 
> so we don't have to pass down the Enrollee/Id into the child factories. This keeps them much lighter weight, and much more importantly, means we can make one certification factory, rather than _one for every Enrollee we want to generate_.
> 
> ```cs
> // somewhere
> public static CertificationFactory = new CertificationFactory();
> 
> // in EnrolleeFactory
> RuleFor(x => x.Certifications, f => CertificationFactory.GenerateBetween(1, 2));
> ```

Yes, there is some extra unnecessary work done by re-using the factories from `prime-dotnet-webapi-tests`.  However, as also discussed in Zoom, if the factories generate lots of mock test data that fulfills our performance testing goals, does it matter?  Why maintain another set of factories for a utility project?